package.json

```diff
@@ -47,6 +47,7 @@
     "eslint-config-finn-prettier": "^3.0.0",
     "in-publish": "^2.0.0",
     "mkdirp": "^0.5.1",
+    "murmurhash3js": "^3.0.1",
```

Comment:
minor: `murmurhash3js` nees to be a dependency and not a devDependecy.

Reply:
fixed via https://github.com/Unleash/unleash-client-node/commit/e56d9c5d59ac37983ae8d410fb86338bd7efdcb9

Reply:
Whoops, thanks for the correction!